### ISSUE TITLE
Instant Search: text description missing for post type icons

### ISSUE THREAD

**Issue by bluefuton** [Automattic/jetpack#25138]
### Impacted plugin

Jetpack, Search

### Steps to Reproduce

1. On a site with Jetpack Instant Search enabled, perform a search with `/?s=&result_format=minimal`.
2. Inspect the post type icons with your browser a11y tools.

### A clear and concise description of what you expected to happen.

All icons should have a text description available.

### What actually happened

The page icon in my results did not have a text alternative.

### Browser

Mozilla Firefox

### Other information

<img width="357" alt="Screen Shot 2022-07-20 at 15 14 09" src="https://user-images.githubusercontent.com/17325/179892607-1fbfac04-d3a8-44ba-89f9-d070a99ec008.png">
<img width="612" alt="Screen Shot 2022-07-20 at 15 48 32" src="https://user-images.githubusercontent.com/17325/179892637-db67a866-1fc4-4c06-9df9-7d2ba7dd3bee.png">

### Reproducibility

Consistent
